src/pages/Header/header.scss

```
@@ -0,0 +1,41 @@
+@mixin flexCenterBox($JCdefault: center, $AIdefault: center) {
+  display: flex;
+  justify-content: $JCdefault;
+  align-items: $AIdefault;
+}
+
+.layoutHeader {
+  @include flexCenterBox(center, start);
+  .headerFlexBox {
+    width: 1000px;
+    .authFlexBox {
+      @include flexCenterBox(flex-end, start);
+      width: 1000px;
+      .layoutAuth {
+        @include flexCenterBox(start, center);
+        height: 38px;
+        li {
+          margin-right: 20px;
+        }
+      }
+    }
+    .navFlexBox {
+      @include flexCenterBox(space-evenly, center);
+      span {
+        padding: 33px 0;
+      }
+    }
+    .layoutNav {
+      display: flex;
+      li {
+        margin-right: 20px;
+      }
+    }
+    .navIconWrap {
+      display: flex;
+      a {
+        padding: 0 10px;
+      }
+    }
+  }
```

```suggestion
.layoutHeader {
  @include flexCenterBox(center, start);
  
  .headerFlexBox {
    width: 1000px;
    
    .authFlexBox {
      @include flexCenterBox(flex-end, start);
      width: 1000px;
      
      .layoutAuth {
        @include flexCenterBox(start, center);
        height: 38px;
        li {
          margin-right: 20px;
        }
      }
    }
    
    .navFlexBox {
      @include flexCenterBox(space-evenly, center);
      span {
        padding: 33px 0;
      }
    }
    
    .layoutNav {
      display: flex;
      li {
        margin-right: 20px;
      }
    }
    
    .navIconWrap {
      display: flex;
      a {
        padding: 0 10px;
      }
    }
  }
```
가독성을 위해 셀럭터 시작할 때는 한 줄 띄워주시기 바랍니다.

---

넵! push하기 전에 한 번 더 확인하도록 하겠습니다!